lib/media/streaming_engine.js

```diff
@@ -1400,7 +1400,7 @@ shaka.media.StreamingEngine = class {
 
     if (mediaState.segmentPrefetch && mediaState.segmentIterator &&
         !this.audioPrefetchMap_.has(mediaState.stream)) {
-      mediaState.segmentPrefetch.evict(presentationTime);
```

Comment:
At first, I thought this would be needed for an instant switching but since this specifically checks that the stream isn't in the audioPrefetchMap, and we have an evict(prefentationTime) elsewhere, 👍🏻